src/language-js/printer-estree.js

```diff
@@ -4030,7 +4040,9 @@ function printArgumentsList(path, options, print) {
           shouldGroupFirst
             ? concat([
                 "(",
+                // @ts-ignore [TODO: ensure printedExpanded is defined]
```

Comment:
I think we should consider resolving this JSDoc type error (and the one below), rather than simply ignoring it.

Looking through the code, I think this should not be a problem but would rather be 100% sure.